src/modules/Providers/views/details/tabs/Networks/ProviderNetworks.tsx

```diff
@@ -26,6 +26,7 @@ type ProviderNetworksProps = {
   loadError?: unknown;
 };
 
+// eslint-disable-next-line no-underscore-dangle
```

Comment:
why leaving it like that? and not removing the unneeded component?

Reply:
Since it's the old code. I just fixed it to pass the eslint errors till refactoring the networks tab.